src/main/java/ti4/buttons/handlers/agenda/AgendaResolveButtonHandler.java

```diff
@@ -1061,13 +1061,47 @@ public static void resolveAgenda(Game game, String buttonID, ButtonInteractionEv
         if (!game.isOmegaPhaseMode()) {
             buttons.add(Buttons.green("proceed_to_strategy", "Proceed to Strategy Phase (will run agenda cleanup and ping speaker)"));
         } else {
-            Button electVoiceOfTheCouncil = Buttons.green("elect_voice_of_the_council", "Elect Voice of the Council");
+            Button proceedToScoring;
+            Button electVoiceOfTheCouncil;
+            String currentVotC = game.getLawsInfo().get(Constants.VOICE_OF_THE_COUNCIL_ID);
+            boolean mustElectVoice = currentVotC == null || currentVotC.isEmpty();
+            String electVoiceText = "Elect Voice of the Council";
+            if (mustElectVoice) {
+                electVoiceText += " (required before scoring)";
+            } else {
+                electVoiceText += " (optional)";
+            }
+            if (aCount < 3) {
+                electVoiceOfTheCouncil = Buttons.gray("elect_voice_of_the_council", electVoiceText);
+                proceedToScoring = Buttons.gray("proceed_to_scoring", "Proceed to scoring objectives");
+            } else {
+                electVoiceOfTheCouncil = Buttons.green("elect_voice_of_the_council", electVoiceText);
+                if (mustElectVoice) {
+                    proceedToScoring = Buttons.gray("proceed_to_scoring", "Proceed to scoring objectives");
+                } else {
+                    String speakerUserID = game.getSpeakerUserID();
+                    Player speaker = game.getPlayer(speakerUserID);
+                    if (speaker != null) {
+                        MessageChannel speakerCardsInfoThread = speaker.getCardsInfoThread();
+                        MessageHelper.sendMessageToChannel(speakerCardsInfoThread, "These are the current votes available for the Voice of the Council vote.");
+                        AgendaHelper.listVoteCount(game, speakerCardsInfoThread);
+                    }
+                    proceedToScoring = Buttons.green("proceed_to_scoring", "Proceed to scoring objectives");
+                }
+            }
```

Comment:
In Omega Phase, the Voice of the Council agenda comes up as a third agenda every round. Voting on it may be optional or required. This changes the "Elect Voice of the Council" and the "Proceed to Scoring" button colors to use gray or green, to guide players to the right button for the situation.

If before the 3rd agenda, "Elect VotC" and "Proceed to Scoring" are gray.
If 3rd agenda or later AND players must vote on VotC, then "Elect VotC" is green and "Proceed to Scoring" is gray.
If 3rd agenda or later and VotC is optional, then both buttons are green.